lib/environments/meteorEnvironment.js

```diff
@@ -321,13 +321,7 @@ export default {
     ['.html', '.css'].forEach((ext: string) => {
       const moduleSpecifier = `${basePath}${ext}`;
       if (fs.existsSync(path.join(config.workingDirectory, moduleSpecifier))) {
-        if (config.get('useRelativePaths')) {
-          moduleSpecifiers.push('.'.concat(path.sep, path.basename(moduleSpecifier)));
-        } else {
-          // Strip the leading '.' off of the moduleSpecifier to turn it into a Meteor compliant
-          // absolute path.
-          moduleSpecifiers.push(moduleSpecifier.slice(1));
```

Comment:
I concur heartily with the change in order. I reviewed quite a few configuration implementations in npm about a month ago and they all went in that direction - for example [node-config's order documentation is here](https://github.com/lorenwest/node-config/wiki/Configuration-Files).

Most also supported many file formats and even allowed ENV and command-line overrides as the last level considered, but I understand the reasons that without a 3rd party package managing this we might want to fall back to just js.

I also concur with having some system in which a function at any level can choose to either inherit or supercede what was produced by the previous level. Many packages seem to have functionality like that.